server/tinylicious/src/resourcesFactory.ts

```diff
@@ -61,7 +61,11 @@ export class TinyliciousResourcesFactory implements IResourcesFactory<Tinyliciou
                 return new Historian(url, false, false);
             },
             winston,
-            undefined /* serviceConfiguration */,
```

Comment:
@DLehenbauer could it be done any differently? Updating config.json in tinylicious has no effect